Type inference for dayjs.utc

Got an error when using this library in a TypeScript project:

**Current Behavior**

```
ERROR in node_modules/@arkecosystem/platform-sdk/dist/utils/dayjs.d.ts:2:40 - error TS2339: Property 'utc' does not exist on type 'typeof dayjs'.

2 export declare const dayjs: typeof day.utc;
```

The TS compiler does not handle the prototype change by the plugin:
https://unpkg.com/browse/@arkecosystem/platform-sdk@0.3.6/dist/utils/dayjs.d.ts


**Possible Solution**

Importing the type explicitly in `tsconfig.json` solved it on the client:

```json
{
	"extends": "./tsconfig.json",
	"include": [
		"node_modules/dayjs/plugin/utc.d.ts"
	]
}

```

--- a/packages/platform-sdk/src/utils/dayjs.ts
+++ b/packages/platform-sdk/src/utils/dayjs.ts
@@ -2,6 +2,7 @@ import day from "dayjs";
 import advancedFormat from "dayjs/plugin/advancedFormat";
 import localizedFormat from "dayjs/plugin/localizedFormat";
 import utc from "dayjs/plugin/utc";
+import "dayjs/plugin/utc";
 
 day.extend(advancedFormat);
 day.extend(localizedFormat);

